### ISSUE TITLE
Support <select multiple> with Control class

### ISSUE THREAD

**Issue by martinsik** [angular/angular#4427]
Hey guys, I'm trying to use `<select>` field with multiple selectable options, in the same way as its documented in [select_control_value_accessor.ts](https://github.com/angular/angular/blob/master/modules/angular2/src/core/forms/directives/select_control_value_accessor.ts) (although the doc block is maybe slightly outdated) eg.:

```html
<select multiple [ng-form-control]="control">
    <option *ng-for="#c of cities" [value]="c"></option>
</select>
```

But when I want to get selected values with `control.value` it returns only the first option selected. According to [writeValue()](https://github.com/angular/angular/blob/master/modules/angular2/src/core/forms/directives/select_control_value_accessor.ts#L56) it doesn't seem to be supported right now.

Is this a planned feature or it's not going to be supported at all? If it's planned I can have a look at it.

**Comment by ocombe** [comment]
Same problem here. Did you find a way to bypass that problem @martinsik ?

**Comment by martinsik** [comment]
Hi @ocombe, I used a little hack. I didn't use Angular's `[ng-control]` directive to control the `<select>` element but I added `(^change)` event that updates the `Control` object associated with this field using `control.updateValue()` to whatever I want although I had to use custom `*ng-if` to mark `<option>`s as `selected="selected"` which makes it a little messy.

**Comment by liulnn** [comment]
How to resolve this problem?

**Comment by fluyy** [comment]
I met the same problem. But I found the elements I selected in event.target.selectedOptions

this is my code, it's work well.

 ···
<select multiple class="form-control multiple-select" size="15" (change)="srcValueChange($event)">
    <option *ngFor="#item of src_items" value={{item}} >{{item}}</option>
</select>
···

and 

```
	srcValueChange(event){
		this.seleted_src_item=[];
		for(var i in event.target.selectedOptions){
			if(event.target.selectedOptions[i].label){
				this.seleted_src_item.push(event.target.selectedOptions[i].label);
			}
		}
	}
```
Is there any other way to solve this problem?

**Comment by aciccarello** [comment]
Just curious. was this fixed by #8069?

**Comment by pkozlowski-opensource** [comment]
Yes, it was, thnx @aciccarello
